manual/owner.md

```diff
@@ -0,0 +1,233 @@
+
+#### Creating optional fields
+
+You can create fields that are hidden or shown to the applicant depending on their answers.
+
+For example, you can create an option list “Do you want to add attachments?” Yes - No. If the applicant chooses “Yes” the attachment field is shown to the applicant. If they choose “No” they will not see the attachment field.
+
+:exclamation: Note! Optional fields only work with Option list.
+
+1. Create an **Option list** and fill in the options
+![Option list](img/optionlist1.png) ![Option list](img/optionlist2.png)
+2. Create another field of any field type.
+![Option list](img/optionlist3.png)
+3. In the another field, select Visible>Only if
+![Option list](img/optionlist4.png)
+4. Field: Locate the Option list you created earlier from the drop-down list
+5. Has value: Select when you want the list to be shown. In this example, Field 2 "Add one or multiple attachments" is shown if the applicant chooses Yes-option in the Field 1. "Do you want to add attachments?".  
+![Option list](img/optionlist5.png)
+
+You can test how the fields work on the Preview-section on the right of the page
+
+### 2. Create a workflow
+
+You can choose between two different workflows: A dynamic workflow and a decider workflow. Choose the one which suits your organisation best. Remember to give your workflow a descriptive name so it will be easier for you to find it later.
+
+**A dynamic workflow** is the default. In dynamic workflow the order of events is flexible and handlers are in charge of approving or rejecting applications. In addition, there can be reviewers and deciders.
+
+**A decider workflow** is a restricted version of the dynamic workflow. Handlers do not have the authority to approve or reject applications. They have to request a separate decider to approve or reject the application. A decider workflow is useful for example in governmental office use.
+
+Assign handlers for a workflow by searching the user by their username or locating them from the drop-down list. We recommend **assigning more than one handler to a workflow** to make sure there are enough handlers to process all the applications.
+
+![Handlers](img/handlers.png)
+
+:exclamation: Note! You can **only find handlers that have previously logged in to REMS or have been added using the API**. If you cannot find the handler you were looking for, please ask them to log in to REMS and try again.
+
+:point_up: Pro tip: It is also possible to make approving and rejecting automatic by selecting **Approver Bot** and **Rejecter Bot** as handlers. Approver Bot will automatically **approve all applications  where all the required fields have been filled**.  Rejecter Bot will reject the application if the applicant or any of the members attached to the application are on the blacklist.
```

Comment:
should we link to docs/bots.md from here?

Reply:
Added link.